WcaOnRails/app/inputs/date_picker_input.rb

```diff
@@ -3,10 +3,10 @@ def input(wrapper_options)
     set_html_options
     set_value_html_option
 
-    template.content_tag :div, class: 'input-group date datetimepicker' do
+    #template.content_tag :div, class: 'input-group date datetimepicker' do
       input = super(wrapper_options) # leave StringInput do the real rendering
-      input_button + input + utc_addon
-    end
+    #  input_button + input + utc_addon
```

Comment:
Because we no longer have a div wrapping the input, the UTC thing is the same width as the page. Also the datepicker doesn't work :P
(The input-with-no-button doesn't work if you wrap it with something, so...)
![utc](https://cloud.githubusercontent.com/assets/1881933/12811915/8c704a04-cb16-11e5-983f-3a5279b6a032.png)

There is a note saying times are in UTC just below...
Do we use this somewhere else other than registration open/close?
![utc2](https://cloud.githubusercontent.com/assets/1881933/12811933/b82a174c-cb16-11e5-8284-6d78adb096ec.png)

I'm commiting a "solution": adding a similiar note to the registration open/close fields.